documentation/docs/introduction.md

```diff
@@ -19,47 +19,56 @@ In this guide, we will go through the most important concepts that developers wi
 
 ## Overview
```

Comment:
The preceding sentence claims that the guide is "programming language agnostic", which is not true, and references "Chapter 4", which I think has changed now. Probably best to remove the last sentence in that paragraph altogether.

https://github.com/iotaledger/identity.rs/blob/cae710f69ac24f0cb8f036e198e126fec87ca22a/documentation/docs/introduction.md?plain=1#L18